.github/workflows/pr.yml

```diff
@@ -15,19 +15,18 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: '1.20'
       - name: lint
         uses: golangci/golangci-lint-action@v3.4.0
         with:
-          version: v1.50.1
+          version: v1.52.2
 
   tests-on-unix:
     needs: golangci-lint # run after golangci-lint action to not produce duplicated errors
     runs-on: ubuntu-latest
     strategy:
       matrix:
         golang:
-          - "1.19"
```

Comment:
The only controversial change here. I do not see benefit of us verifying under 2 versions of golang. 
We will likely to introduce another version soon so for GA we should have CI/CD process on single version for simplicty. 

Linting, formatting, introspections are all version based.

Reply:
should we remove the matrix then?

Reply:
Good point. We can always restore it for 1.21

Reply:
if we intend to restore it, there is no need to remove